src/styles/pages/demo.scss

```diff
@@ -1,9 +1,25 @@
 .demo-page {
-  background-color: #F7F7F7;
+  background-color: #f7f7f7;
+  color: #000000;
   min-height: 100vh;
-  font-family: 'Roboto';
+  font-family: "Roboto";
 
+  .bold {
```

Comment:
`bold`, `visible`, and `hidden` seem like useful utility classes to have in general. Let's move them into their own file and put `!important` at the end so that we can use them elsewhere and have the specificity rules make them always apply.

Reply:
Suggestions on where to move them?  global.scss?